### PR TITLE
Allow to pass service name for initializing FTLib consensus

### PR DESCRIPTION
**Pull request by terrytangyuan**
Part of https://github.com/sql-machine-learning/elasticdl/issues/1710.

Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

File: elasticdl/python/collective_ops/communicator.py [hunk 1]
```diff
@@ -16,10 +18,15 @@
 
 
 class CollectiveCommunicator(object):
-    def __init__(self):
+    def __init__(self, svc_name=None):
```
Comment (skydoorkai) [inline elasticdl/python/collective_ops/communicator.py:21]: What is this `svc_name`? Current worker's service name, or FtLib service?

Comment (terrytangyuan) [reply elasticdl/python/collective_ops/communicator.py:21]: FTLib k8s service. I’ll work on adding that service in a follow-up PR

Comment (skydoorkai) [reply elasticdl/python/collective_ops/communicator.py:21]: `getaddrinfo` of this `svc_name` will return a list of all worker ip info?

Comment (terrytangyuan) [reply elasticdl/python/collective_ops/communicator.py:21]: Yes, this way the consensus method knows what addresses are known.

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**